extensions/amp-analytics/0.1/vendors/taboola.json

```diff
@@ -0,0 +1,20 @@
+{
+  "vars": {
+    "aid": ""
+  },
+  "requests": {
+    "url": "https://3p.ampproject.net/custom/amp-analytics-taboola.html?url=${canonicalUrl}&aid=${aid}"
+  },
+  "triggers": {
+    "pageview": {
+      "on": "visible",
+      "request": "url",
+      "iframePing": true
+    }
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
+  }
```

Comment:
amp-analytics handles defaulting from beacon => xhrpost => image. Just making sure that you want `image` requests to always be sent.

Reply:
@micajuine-ho Reviewed the docs again. We dont need any overrides. So removed that section. Thanks

Reply:
@micajuine-ho After testing I need the image GET request